api/src/multiaccessy/accessy.py

```diff
@@ -1,32 +1,36 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 from logging import getLogger
 import os
 import threading
-import uuid
-import warnings
+from typing import Union
 
 import requests
 
-from create_env import config
+from service.config import config
 
 logger = getLogger("accessy")
 
 
+UUID = str  # The UUID used by Accessy is a 16 byte hexadecimal number formatted as 01234567-abcd-abcd-abcd-0123456789ab (i.e. grouped by 4, 2, 2, 2, 6 with dashes in between)
+MSISDN = str  # Standardized number of the form +46123123456
+
 ACCESSY_URL = config.get("ACCESSY_URL")
 ACCESSY_CLIENT_SECRET = config.get("ACCESSY_CLIENT_SECRET", log_value=False)
 ACCESSY_CLIENT_ID = config.get("ACCESSY_CLIENT_ID")
+ACCESSY_LAB_ACCESS_GROUP = config.get("ACCESSY_LAB_ACCESS_GROUP")
+ACCESSY_SPECIAL_ACCESS_GROUP = config.get("ACCESSY_SPECIAL_ACCESS_GROUP")
 
 
 def check_response_error(response: requests.Response, msg: str = None):
     if not response.ok:
         msg_str = f"\n\tMessage: {msg}" if msg is not None else ""
-        logger.error(f"Got an error in the response. {response.status_code=}{msg_str}")
+        logger.error(f"Got an error in the response for {response.request.path_url}. {response.status_code=}{msg_str}")
         raise RuntimeError(msg)
 
 
 @dataclass
 class AccessySession:
-    session_token: uuid
+    session_token: UUID
 
     @classmethod
     def create_session(cls, client_id: str, client_secret: str) -> "AccessySession":
```

Comment:
Do not check the parameters like this. Makes it hard to make something string like and use that.

Reply:
Removed check